int/api/cmd/execute_function.go

```diff
@@ -39,8 +40,18 @@ func ExecuteFunctionHandler(params operations.CmdExecuteFunctionParams, app *fyn
 	}
 
 	password := gui.AskPassword(wallet.Nickname, app)
```

Comment:
In go you can returned multiples values. The way to handle error in go is very strict : the last returned value should be an error.

This line should be something like:
```suggestion
	clearPassword, err := gui.AskPassword(wallet.Nickname, app)
```